### DISCUSSION TITLE
Help tell which plugin supports text highlighting

### DISCUSSION THREAD

**Discussion by punk2plus** [remarkjs/remark#954] [Q&A] [answered]
like syntax： ==abc==
typora can support this syntax

reference： https://github.com/markdown-it/markdown-it-mark

**Comment by wooorm** [comment, marked as answer]
Generally we recommend not adding non-standard things that don’t work in many places. You can use HTML or directives instead. See https://github.com/micromark/micromark#extensions for everything you need to know about syntax extensions.
